src/main/java/catbot/tasklist/Task.java

```diff
@@ -0,0 +1,56 @@
+package catbot.tasklist;
+
+import catbot.ui.ConsoleColors;
+
+/**
+ * The parent class of all tasks.
+ */
+public abstract class Task {
+    protected boolean isDone;
+    final String description;
```

Comment:
You should make 'description' either protected or private. By default, Java leaves unspecified variables as public and this can violate information hiding